src/components/10-atoms/spinner/pw.ui.test.js

```diff
@@ -0,0 +1,17 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-spinner';
+
+describe('Spinner', () => {
+  it('should render', async () => {
+    await page.goto(
+      `${host}/iframe.html?id=components-spinner--spinner&viewMode=story`
+    );
+    await page.waitForSelector(tag);
+
+    expect(
+      (await page.isVisible('.a-spinner__dot-1')) &&
+        (await page.isVisible('.a-spinner__dot-2')) &&
+        (await page.isVisible('.a-spinner__dot-3'))
+    ).toBeTruthy();
+  });
+});
```

Comment:
You could add a test that validates the following:

- Open spinner, check if it is displayed
- Close the spinner, check that is is not displayed
- Open spinner, check if it is displayed

And maybe another test that validates the colors of the dots.

Reply:
There is currently no option to turn on/off the spinner so it would make less sense to implement a test for it.